lib/smack/SplitStructLoadStore.cpp

```diff
@@ -0,0 +1,148 @@
+#include "smack/SplitStructLoadStore.h"
+#include "llvm/IR/InstIterator.h"
+
+namespace smack {
+
+using namespace llvm;
+
+std::vector<Value*> getFirsts(std::vector<std::pair<Value*, unsigned> > lst)
```

Comment:
You have a coding style mistake here - we put open curly braces after function definition, meaning after `)`. Take a look at other files for examples and then please fix this.